### PR TITLE
WP-9223: fix obfuscate

### PR DESCRIPTION
**Pull request by tomwang-varicent**
app pr: https://github.com/symon-ai/app/pull/6920

create v1.0.0

### CODE REVIEW

File: tap_postgres/db.py [hunk 1]
```diff
@@ -31,19 +33,24 @@ def calculate_destination_stream_name(stream, md_map):
     if include_schemas_in_destination_stream_name:
         return "{}_{}".format(md_map.get((), {}).get('schema-name'), stream['stream'])
 
-    return stream['stream']
+    return stream['tap_stream_id']
```
Comment (kchan-varicent) [inline tap_postgres/db.py:36]: do you need to fix the return above as well?

**Review by kchan-varicent — COMMENTED** [review]
are all these deletes going to cause conflicts in the future when we decide to fetch upstream changes?

File: tap_postgres/db.py (hunk 1 above)
Comment (tomwang-varicent) [reply tap_postgres/db.py:36]: `include_schemas_in_destination_stream_name` is hardcoded to false. Maybe we can remove it if we will never use it.

**Review by kchan-varicent — APPROVED** [review]

File: tap_postgres/db.py (hunk 1 above)
Comment (tomwang-varicent) [reply tap_postgres/db.py:36]: Should I remove it right now?

Comment (kchan-varicent) [reply tap_postgres/db.py:36]: no that's fine

**Pull request merged**